org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServiceAccessor.java

```diff
@@ -351,13 +351,13 @@ protected static Collection<LanguageServerWrapper> getLSWrappers(@NonNull final
 	 */
 	@NonNull
 	public static LanguageServerWrapper getLSWrapper(@Nullable IProject project,
-			@NonNull LanguageServerDefinition serverDefinition) throws IOException {
-		return getLSWrapper(project, serverDefinition, null);
+			@NonNull LanguageServerDefinition serverDefinition, @Nullable URI initialUri) throws IOException {
```

Comment:
Instead of adding an additional parameter here which breaks backwards compatibility I would appreciate if instead a new method can be added with the additional parameter. That would also make all these `getLSWrapper(.., null)` changes in this commit unnecessary.